src/js/call.js

```diff
@@ -152,7 +152,7 @@ Call.parseCandidate = function(text) {
   };
 };
 
```

Comment:
Define "Call.origResponse_ = null" here and comment what the static variable does.
Btw could you get a more descriptive name? Call.cachedIceServers_?